Userland/Utilities/pwd.cpp

```diff
@@ -11,7 +11,7 @@
 
 ErrorOr<int> serenity_main(Main::Arguments)
 {
-    TRY(Core::System::pledge("stdio"));
+    TRY(Core::System::pledge("rpath stdio"));
```

Comment:
Commit message nitpick: mind being a bit more destrictive about what you're fixing? a-la "Add missing rpath pledge"

Reply:
The commit message has been updated.